src/components/List.js

```diff
@@ -0,0 +1,11 @@
+function List({ items, renderItem }) {
+    return (
+        <ul>
+            {items.map((item, index) => (
+                <li key={index}>{renderItem(item)}</li>
+            ))}
+        </ul>
+    );
+}
+
+export default List;
```

Comment:
I think this is a good use case for generic components if you think you'll be having multiple different kinds of lists in the future!